src/patterns/start-pages/index.md.njk

```diff
@@ -1,41 +1,72 @@
 ---
-title: Start pages
-description: A starting point for your digital service on GOV.UK
+title: Start using a service
+description: Create a starting point for your digital service on GOV.UK
 section: Patterns
-theme: Pages
-aliases:
+theme: Help users to…
+aliases: start page, start pages
 backlog_issue_id: 111
 layout: layout-pane.njk
 ---
 
-{% from "_example.njk" import example %}
+GOV.UK services must start on a GOV.UK content page. This links your service to the rest of GOV.UK, and means that users can find it using search or through GOV.UK’s navigation.
 
-Use this pattern to test a start page in your prototype.
+## When to use this pattern
 
-To get a start page for your public beta or live service, you need to [contact the GDS content team](https://www.gov.uk/service-manual/service-assessments/get-your-service-on-govuk) before your alpha assessment.
+Use this pattern to help you prototype a start point for your service, so you can test as much of the user journey as possible.
 
-{{ example({group: "patterns", item: "start-pages", example: "default", html: true, nunjucks: true, open: false}) }}
+The public-facing version of your service will need to start on a GOV.UK content page, published using one of the GOV.UK publishing apps. You’ll usually need to agree content for this GOV.UK content page with the GDS content team. Read [guidance on how to get your service on GOV.UK](https://www.gov.uk/service-manual/service-assessments/get-your-service-on-govuk).
 
-## When to use this pattern
+## How it works
 
-If your service is in public beta or live, you must have a starting point hosted on GOV.UK. This will link your service to the rest of GOV.UK.
+A service start point should:
+- give the user just enough information to help them understand what the service does and whether it will meet their need - including [giving the service an name that reflects the problem it solves for users](https://www.gov.uk/service-manual/design/naming-your-service)
```

Comment:
```suggestion
- give the user just enough information to help them understand what the service does and whether it will meet their need - including [giving the service a name that reflects the problem it solves for users](https://www.gov.uk/service-manual/design/naming-your-service)
```